core/modules/user/user.module

```diff
@@ -1630,6 +1635,18 @@ function user_menu_alter(&$items) {
   }
 }
 
+/**
+ * Return the login page appearance setting.
+ */
+function user_login_appearance_setting() {
+  $appearance_setting = config_get('system.core', 'user_login_appearance_setting');
+  // If Layout module is disabled, use links.
+  if ($appearance_setting == 'simplified' && !function_exists('layout_suppress')) {
```

Comment:
I wonder, though, when the Layout module can be disabled. It's a _required_ module in B.

And the check function_exists() instead of module_exists() makes me wonder even more, if the check in this form tries to hide a problem caused by something else...

Reply:
Actually `testMainContentFallback()` disables Layout module and was failing with a function not found exception without this. Didnt think it was worth the overhead of `module_exists()`.